syft/common/util.py

```diff
@@ -3,6 +3,13 @@
 
 import torch
 import numpy as np
+from enum import Enum
+
+
+class TranslationTargets(Enum):
+    PYTORCH = 'torch'
+    TENSORFLOW = 'tf'
+    TENSORFLOW_JS = 'tfjs'
```

Comment:
Since translation is limited to `Plans` (at least for now), let's put this somewhere in the `execution/translation` package. It could go in the `__init__.py` for now.